src/cic/model.clj

```diff
@@ -211,3 +211,16 @@
                 placements
                 (let [next-placement (phase-transition (zero? offset) age placement)]
                   (recur next-offset next-placement (conj placements {:offset next-offset :placement next-placement})))))))))))
+
+(defn joiner-birthday-model
+  "Accepts quantiles for age zero joiner ages in days and returns a birthday-generating model"
```

Comment:
Is this b/c the probabilities around 0 age children are different from the other ages? It would be good to have a DSDR for this that is referenced here (as we'll forget). A FIXME comment with the detail should work for now